### PR TITLE
 [zh]  sync add external-loadbalancer-support.md

### PR DESCRIPTION
**Pull request by jiecloud**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/boilerplates/external-loadbalancer-support.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+---
+{{<警告>}}}
```
Comment (windsonsea) [inline content/zh/boilerplates/external-loadbalancer-support.md:3]:
```suggestion
{{< warning >}}
```

File: content/zh/boilerplates/external-loadbalancer-support.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+这些说明假定你的Kubernetes集群支持外部负载均衡器（即`LoadBalancer`类型的服务）。
```
Comment (windsonsea) [inline content/zh/boilerplates/external-loadbalancer-support.md:4]:
```suggestion
这些说明假定您的 Kubernetes 集群支持外部负载均衡器（即 `LoadBalancer` 类型的服务）。
```

File: content/zh/boilerplates/external-loadbalancer-support.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+请参考 [ingress 控制](/zh/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports)了解详情。
```
Comment (windsonsea) [inline content/zh/boilerplates/external-loadbalancer-support.md:5]:
```suggestion
请参考 [Ingress 控制](/zh/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports)了解详情。
```

File: content/zh/boilerplates/external-loadbalancer-support.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+{{</warning >}}
```
Comment (windsonsea) [inline content/zh/boilerplates/external-loadbalancer-support.md:6]:
```suggestion
{{< /warning >}}
```

**Comment by windsonsea** [comment]
如果已按 comments 做了修改，请点击 Resolve conversation，让每位 reviewer 知道进度。

如果你对 comments 有异议或部分接受，请给出反馈。

**Comment by jiecloud** [comment]
> 如果已按 comments 做了修改，请点击 Resolve conversation，让每位 reviewer 知道进度。
> 
> 如果你对 comments 有异议或部分接受，请给出反馈。
> 
> 中英之间需要留一个空格，如果嫌麻烦，可以试试 [prettier](https://prettier.io/docs/en/install.html)

学习了，多谢社区大佬的建议。

**Review by windsonsea — APPROVED** [review]
Let's kick this in though it includes some nits.

**Pull request merged**